src/frontc/cparser.mly

```diff
@@ -233,12 +233,25 @@ let transformOffsetOf (speclist, dtype) member =
   let resultExpr = CAST (sizeofType, SINGLE_INIT addrExpr) in
   resultExpr
 
+  let queue_to_int64_list queue =
+    List.rev (Queue.fold (fun l e -> List.rev_append e l) [] queue)
+
+  let queue_to_string queue =
+    let buffer = Buffer.create (Queue.length queue) in
+    Queue.iter
+      (List.iter
+        (fun value ->
+          let char = int64_to_char value in
+          Buffer.add_char buffer char))
+      queue;
+    Buffer.contents buffer
+
```

Comment:
Using queues inside lexer rules, this is some insanity. But it was already here before, so let it be.